kyo-core/shared/src/test/scala/kyo/IOTest.scala

```diff
@@ -109,9 +109,6 @@ class IOTest extends Test:
             }
             succeed
         }
-        "doesn't accept other pending effects" in {
-            assertDoesNotCompile("IO.Unsafe.run[Int < Options](Options.get(Some(1)))")
```

Comment:
This is a good example of how `assertDoesNotCompile` generates tech debt. `Options` doesn't exist for a long time now.